### ISSUE TITLE
Joyride creates duplicated DOM elements when re-running the tour

### ISSUE THREAD

**Issue by v3rron** [gilbarbara/react-joyride#777]
## 🐛 Bug Report

Package Version: 2.3.2

Hi, first of all, thanks for the great package. 
Please see the bug described below and Code Sandbox example with reproduction.

When implementing Joyride  in controlled mode, the DOM is being duplicated on recurring runs, instead of being cleared properly.

## To Reproduce

1. Go to Code Sandbox link below
2. Click on Start Tour button
3. Go through all the steps
4. Click on Start Tour button again
5. Check HTML DOM in DevTools

## Expected behavior

The DOM should be cleared whenever the tour ends or being skipped.

![Screen Shot 2022-01-27 at 9 52 31 PM](https://user-images.githubusercontent.com/1641115/151479069-4334b5e7-cfc1-4450-9726-400168db35b4.png)

## Link to repl or repo (highly encouraged)

[Code Sandbox Link](https://codesandbox.io/s/react-joyride-dom-bug-zqcxd)

Here's the integration code, just in case:
```javascript
import Joyride, { STATUS } from "react-joyride";
import { useState } from "react";

const JOYRIDE_STEPS = [
  {
    target: "#my-first-step",
    content: "This is my awesome feature!"
  },
  {
    target: "#my-second-step",
    content: "This is another awesome feature!"
  },
  {
    target: "#my-third-step",
    content: "Now this feature is the best!"
  }
];

export default function App() {
  const [showTour, setShowTour] = useState(true);

  const handleCallback = ({ status }) => {
    if ([STATUS.SKIPPED, STATUS.FINISHED].includes(status)) {
      console.log("Tour finished or skipped");
      setShowTour(false);
    }
  };

  return (
    <div className="App">
      <h1>Hello CodeSandbox</h1>
      <h2>Start editing to see some magic happen!</h2>
      <button onClick={() => setShowTour(true)}>SHOW TOUR</button>
      <h3 id="my-first-step">First Step</h3>
      <h4 id="my-second-step">Second Step</h4>
      <h5 id="my-third-step">Third Step</h5>
      <Joyride
        run={showTour}
        steps={JOYRIDE_STEPS}
        callback={handleCallback}
        continuous
      />
    </div>
  );
}
```

**Comment by wuifdesign** [comment]
having the same problem. any solution for this?

**Comment by gilbarbara** [comment]
Fixed in 2.5.0
